convert.py

```diff
@@ -79,13 +81,18 @@ def get_page(mt_export_file):
                 line_buffer.append(line)
 
 
-def process_file():
-    for page in get_page(sys.argv[1]):
+def process_file(filename):
+    for page in get_page(filename):
         Page(page).convert_to_markdown()
 
 
 if __name__ == '__main__':
-    if (sys.argv[1]):
-        process_file()
-    else:
-        print("Please pass in file")
+    while True:
+        filename = input("Please enter a file name: ")
+        try:
+            process_file(filename)
+        except OSError:
+            print("Huh, didn't catch it. Try one more time. Thanks.")
+        except (KeyboardInterrupt, EOFError):
+            sys.exc_clear()
+            pass
```

Comment:
I think this `pass` is unnecessary here - or is there a reason for it? Either way, really not a big deal.